tasty-query/shared/src/main/scala/tastyquery/reader/pickles/PickleReader.scala

```diff
@@ -692,6 +736,8 @@ private[pickles] class PickleReader {
 
 private[reader] object PickleReader {
 
+  private val MangledDefaultGetterNameRegex = "^(.+)\\$default\\$([0-9]+)$".r
```

Comment:
Does this work with default constructor arguments?  Maybe add a test for those.

Reply:
It works. I added a test :)